aws/modules_common.tf

```diff
@@ -35,6 +35,7 @@ module "codebuild_cd_cid-checker" {
   git_repository_name      = "cid-checker"
   get_global_configuration = local.make_codebuild_global_configuration
   privileged_mode          = true
+  codebuild_image          = "docker:dind"
```

Comment:
What is this variable that is not used anywhere else in the PR?

Reply:
it is a variable that uses inside a codebuild module.